scripts/contentScript.js

```diff
@@ -212,6 +221,46 @@ async function showFloatingActionButton(orgName, orgRepo) {
 	}
 }
 
+// showing the important files in a pr
+async function showImpFileInPr(repoOwner,repoName,userId) {
+	const body = {
+		"repo_owner": repoOwner,
+		"repo_name": repoName,
+		"user_id": userId,
+		"is_github": true
+	}
+	let hashList = await apiCall(url,body);
```

Comment:
And why is this not `const`? `hashList` is not a good name. I suggest `encryptedFileNames`.

```suggestion
	const encryptedFileNames = await apiCall(url,body);
```

Reply:
Done